cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -97,6 +97,18 @@ def generate_video_file(filename, width=1280, height=720, duration=1, fps=25, co
     return [(width, height)] * total_frames, f
 
 
+def compare_datasets(test, expected, actual):
```

Comment:
This function needs some explanation. Why can't we just use `dm_compare_datasets`?

Reply:
dm_compare_datasets fails if attributes are not strictly equal, but export/import leads to slight fluctuations in "rotation" attribute, therefore dm_compare_datasets can not be used to compare it

Reply:
added a comment about it